src/components/app.test.js

```diff
@@ -8,32 +8,41 @@ it('renders without crashing', () => {
   shallow(<App />);
 });
 
-it('displays default content', () => {
-  const div = document.createElement('div')
-  render((
-    <MemoryRouter>
-      <App />
-    </MemoryRouter>
-  ), div)
-  console.assert(div.innerHTML.match(/Easily submit and score QPP data in real-time via API/))
-})
+const contentTestStrings = {
+  '/': 'Easily submit and score QPP data in real-time via API',
+  '/developer-preview': 'Developer Preview',
+  '/tutorial': 'API Tutorial',
+  '/submission': 'The Submissions resource represents one year of performance data'
+}
 
```

Comment:
Not sure how exhaustive and verbose we want tests to be, treating this as kind of a sampling